frontend/packages/kubevirt-plugin/src/components/dashboards-page/vm-dashboard/queries.ts

```diff
@@ -10,28 +10,29 @@ export enum VMQueries {
 }
 
 const queries = {
+  // We don't set namespace explicitly in the PromQL template because it is
+  // being injected anyway by prom-label-proxy when we query Thanos.
```

Comment:
fwiw you could potentially still leave it here for clarity, although being a no-op.

Reply:
I don't mind adding it, but I think that it causes confusion where you specify the namespace in the query and it doesn't really being used... especially in cases where there will be a conflict between what is rendered here and what is present in the query param...

Reply:
sounds good :+1: